cmd/cape/cmd/list.go

```diff
@@ -66,6 +73,17 @@ func doList(url string, insecure bool, auth entities.FunctionAuth) error { //nol
 		return errors.New("list failed")
 	}
 
-	// TODO print list here
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return errors.New("could not read response body")
+	}
+
+	var deploymentNames []DeploymentName
+	err = json.Unmarshal(body, &deploymentNames)
+	if err != nil {
+		return errors.New("malformed body in response")
+	}
+
+	fmt.Printf("%v", deploymentNames)
```

Comment:
we could also loop through the deploymentNames and print them out so that it would look something like this:
```
function_name_1
function_name_2
function_name_3
```

Reply:
or a form of a table! think that was done previously for List

Reply:
Added a table! It does look much nicer 💯